python_binding/lsm/_client.py

```diff
@@ -1427,3 +1427,320 @@ def volume_ident_led_clear(self, volume, flags=FLAG_RSVD):
+                The policy for read cache. Valid values are:
+                    * lsm.Volume.READ_CACHE_POLICY_ENABLED
+                        Read cache is enabled, when reading I/O on previous
+                        unchanged written I/O or read I/O in cache will be
+                        return I/O initiator immediately without checking
+                        backing store(normally disk).
+                    * lsm.Volume.READ_CACHE_POLICY_DISABLED
+                        Read cache is disabled.
+                    * lsm.Volume.READ_CACHE_POLICY_UNKNOWN
+                        Plugin failed to detect the read cache policy.
+            read_cache_status (int)
+                The status of read cache. Valid values are:
+                    * lsm.Volume.READ_CACHE_STATUS_ENABLED
+                    * lsm.Volume.READ_CACHE_STATUS_DISABLED
+                    * lsm.Volume.READ_CACHE_STATUS_UNKNOWN
+            physical_disk_cache (int)
+                Whether physical disk's cache is enabled or not.
+                Please be advised, HDD's physical disk ram cache might be not
+                protected by storage system's battery or capacitor on sudden
+                power loss, you could lose data if a power failure occurs during
+                a write process.
+                For SSD's physical disk cache, please check with the vendor of
+                your hardware RAID card and SSD disk.
+                Valid values are:
+                    * lsm.Volume.PHYSICAL_DISK_CACHE_ENABLED
+                        Physical disk cache enabled.
+                    * lsm.Volume.PHYSICAL_DISK_CACHE_DISABLED
+                        Physical disk cache disabled.
+                    * lsm.Volume.PHYSICAL_DISK_CACHE_USE_DISK_SETTING
+                        Physical disk cache is determined by the disk vendor
+                        vis physical disks' SCSI caching mode page(0x08 page).
+                        It's strongly suggest to change this value to
+                        lsm.Volume.PHYSICAL_DISK_CACHE_ENABLED or
+                        lsm.Volume.PHYSICAL_DISK_CACHE_DISABLED
+                    * lsm.Volume.PHYSICAL_DISK_CACHE_UNKNOWN
+                        Plugin failed to detect the physical disk status.
+        SpecialExceptions:
+            LsmError
+                ErrorNumber.NO_SUPPORT
+                ErrorNumber.NOT_FOUND_VOLUME
+        Capability:
+            lsm.Capabilities.VOLUME_CACHE_INFO
+        """
+        return self._tp.rpc('volume_cache_info', _del_self(locals()))
+
+    @_return_requires(None)
+    def volume_physical_disk_cache_set(self, volume, pdc, flags=FLAG_RSVD):
+        """
+        lsm.Client.volume_physical_disk_cache_set(self, volume, pdc,
+                                                  flags=lsm.Client.FLAG_RSVD)
+
+        Version:
+            1.3
+        Usage:
+            Change the setting of RAM physical disk cache of certain volume.
```

Comment:
+            Change the setting of RAM physical disk cache of the specified volume.
 +            On some products (like HPE SmartArray), this action will be effective
 +            at the system level which means that even though you are requesting a change
 +            on a certain volume, the change will apply to all other volumes on the same
 +            controller(system).